packages/entities/entities-routes/src/components/RouteForm.vue

```diff
@@ -161,12 +150,14 @@ import type {
   TypedRouteRulesPayload,
 } from '../types'
 import {
+  Methods,
```

Comment:
Why can't we directly use it like `RoutingRulesEntities.Methods`?
It makes us confusing with the `Method`(#line144)